docs/changelog/breaking-changes.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta
+  title="Changelog/Upcoming Breaking Changes"
+  parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+# Upcoming Breaking Changes
+
+## `@jobber/components@5.x.x`
+
+### Autocomplete
+
+Removing `inputRef` from `Autocomplete` component. Use `ref` instead.
```

Comment:
If our reasoning behind this is that `inputRef` doesn't work then we can safely assume the other ones that exist on other input doesn't work as well. Otherwise, we risk having more prop inconsistencies since it would still exist on some.

Reply:
Doesn't that defeat the purpose of this? We're saying we want to expose the breaking change we want to do but now you're saying we shouldn't 🤔

Reply:
I think we should expose breaking changes that have been done and are in the version that is on NPM. Mainly so people can make changes ahead of time. For example if we said that this was for every input, and people wanted to fix code ahead of the breaking change then functionality would break because it hasn't been done yet

Reply:
I think this should be for more actionable breaking changes, i.e. things people can fix ahead of time